WcaOnRails/app/views/competitions/index.js.erb

```diff
@@ -2,5 +2,6 @@ history.pushState(null, document.title, <%= current_url.to_json.html_safe %>);
 
 $('#competitions-list').html('<%=j render 'index_competitions_list', competitions: @competitions %>');
 
-$('#search-results').removeClass('competitions-list').removeClass('competitions-map').removeClass('competitions-admin');
-$('#search-results').addClass('competitions-<%= @display %>');
+$('#search-results').removeClass('competitions-list competitions-map competitions-admin').addClass('competitions-<%= @display %>');
+
+$('#competition-query-form').removeClass('list admin map').addClass('<%= @display %>');
```

Comment:
Woah, this code looks very similar to the previous line =)

My gut feeling is that we could squeeze this down to one line by putting this `list/map/admin` class on something higher up in the DOM, but that can be improvement for a different PR if someone feels like it.

I do worry that we're going to forget to add something here when/if a new tab is every added, but I can live with this as is.